rules/core.js

```diff
@@ -0,0 +1,99 @@
+module.exports = {
+  extends: ["stylelint-config-standard", "stylelint-config-rational-order"],
+  plugins: ["stylelint-order", "stylelint-config-rational-order/plugin"],
+  rules: {
+    /* Specify the alphabetical order of properties within declaration blocks. */
+    /* https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-alphabetical-order */
+    "order/properties-alphabetical-order": null,
+    /* Specify the order of properties within declaration blocks. */
```

Comment:
Can we separate each comment + rule with a new line for a better readability?

```
// Specify the alphabetical order of properties within declaration blocks.
// https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-alphabetical-order
"order/properties-alphabetical-order": null,

// Specify the order of properties within declaration blocks.
// https://github.com/hudochenkov/stylelint-order/tree/master/rules/properties-order
"order/properties-order": [],
```

(I've also change the comment style. Doesn't make sense to me to use multiline-comments in a single line. Break the line between 80-96 characters)